core/services/validators.go

```diff
@@ -202,14 +202,17 @@ func validateFeeds(feeds models.Feeds, store *store.Store) error {
 			}
 		case map[string]interface{}: // named feed - ex: {"bridge": "bridgeName"}
 			bridgeName := feed["bridge"]
+			bridgeNameString, ok := bridgeName.(string)
```

Comment:
Probably should check `ok` first right?

Reply:
In the three error messages below this, the first two check the key, and the last one checks the value. I think that this order is logical. I don't want to throw the error "unsupported value" just because the key is spelled wrong in the json.

Reply:
Ahh, then you probably want to use the `ok` version of `feed["bridge"]`.